Milestone1

paulzay opened the pull request:
### What does this pr do?
Create an erd of the different entities of the project

paulzay committed:
add erd diagram

paulzay committed:
update readme file

uryelah reviewed (changes requested):
Hi **Juma Juma & Zay Ogolla**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job kicking off another project!
You still need to review a bit your ERD associations. Pause a bit to think how each model connects to the others and read on the relationship notations to use the correct ones.
At the end your ERD should look similar to[ this one.](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines/-/raw/master/images/rails/final_project/erd.png)

Check the pointers below and keep on coding! 🏃 💨 

# Required Changes 🏗️ 🐛 💡
- [ ] The Friendship table should receive two different arrows coming from User if both the user and friends come from the users model and friendship belongs to both.

- [ ] Comments should belong to a User and the user can have many comments. Add the missing association between these models.

- [ ] A Post can have many comments but a Comment should belong to only one Post and also belong to one User. Fix the Post/Comment association.

- [ ] The Like should belong to a User and also either belong to one Comment or to one Post. Make sure to add the mission association and make sure the notation to the existing ones is correct. [Check this useful stackoverflow thread about it.](https://stackoverflow.com/questions/4755497/single-line-with-an-arrow-mark-in-entity-relattionship-diagram)
-----

 **Kindly fix the pointed out issues and comments below before requesting the next code review**

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

paulzay committed:
Remodel the erd

1. A user has many posts
2. A user has many comments
3. A post has many comments
4. Comment belongs to one user and one post

rOluochKe reviewed (approved):
Hello Team
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!
